### PR TITLE
CR for #20526 - Fix Mixamo avatar clipping in HMD view

### PR DESCRIPTION
**Pull request by ctrlaltdavid**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 7afd990e2e4ac8c9c29862da4bcf2cea98ac8112 available at: https://gist.github.com/b728c63fc901ef4ac75c<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4771.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4771.dmg

**Comment by cozza13** [comment]
tested and it works.

**Comment by cozza13** [comment]
test please

**Comment by cozza13** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 7afd990e2e4ac8c9c29862da4bcf2cea98ac8112 available at: https://gist.github.com/b728c63fc901ef4ac75c<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4771.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4771.dmg

File: interface/src/avatar/MyAvatar.cpp [hunk 1]
```diff
@@ -1178,14 +1178,8 @@ void MyAvatar::renderBody(ViewFrustum* renderFrustum, RenderArgs::RenderMode ren
         } else {
             float clipDistance = _skeletonModel.getHeadClipDistance();
             if (OculusManager::isConnected()) {
```
Comment (Atlante45) [inline interface/src/avatar/MyAvatar.cpp]: We might want to remove that if altogether.
I've been clipping lately (without Oculus) so I just tried your formula with an `if (true)` and it works, I'm not clipping anymore.

**Comment by hifi-gustavo** [comment]
Build console output for commit: eb438e9ccbd01bcb5ff3e4b295d266b12f8d0b11 available at: https://gist.github.com/b728c63fc901ef4ac75c<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4771.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4771.dmg

**Comment by jherico** [comment]
This looks good to me Clement, and it solves https://app.asana.com/0/26225263936266/32958961688299 from my testing

**Pull request merged**